examples/dialog-modal/datepicker-dialog.html

```diff
@@ -0,0 +1,771 @@
+</head>
+<body>
+  <nav aria-label="Related Links" class="feedback">
+    <ul>
+      <li><a href="../../#browser_and_AT_support">Browser and Assistive Technology Support</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/projects/27">Related Issues</a></li>
+      <li><a href="../../#dialog-modal">Design Pattern</a></li>
+    </ul>
+  </nav>
+  <main>
+    <h1>Date Picker Dialog Example</h1>
+    <p>
+      <strong>NOTE:</strong> This example page is work in progress.
+      Please provide feedback in
+      <a href="https://github.com/w3c/aria-practices/issues/1046">issue 1046.</a>
+    </p>
+    <p>
+      The date picker example below implements the <a href="../../#dialog-modal">dialog design pattern</a>.
+      The date picker dialog box in this example is opened when keyboard focus is moved to the calendar button and a <kbd>space</kbd> or <kbd>return</kbd> key pressed or clicking on the calendar button.
+      The dialog appears on activation of the button and can be closed using the OK or Cancel button.   The date picker dialog uses a <code>grid</code> pattern to show and select a date using the cursor keys.  Additional buttons in the dialog box can be used for changing the month and year shown in the grid.
+    </p>
+  <section>
+    <h2 id="ex_label">Example</h2>
+
+    <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+    <div id="example">
+
+      <div id="myDatepicker"
+           class="datepicker">
+
+        <div class="date">
+          <label for="id-textbox-1">Date</label>
+
+          <input type="text"
+                placeholder="mm/dd/yyyy"
+                id="id-textbox-1"
+                aria-autocomplete="none">
+
+          <button class="icon"
+            aria-label="Calendar"
+            aria-expanded="false"
+            aria-haspopup="dialog"
+            aria-controls="id-datepicker-1"
+            aria-describedby="id-datepicker-1-button-desc"
+            >
+            <span class="fa fa-calendar-alt fa-2x"></span>
+          </button>
+
+        </div>
+
+        <div id="id-datepicker-1"
+             class="datepickerDialog"
+             role="dialog"
+             aria-modal="true"
+             aria-labelledby="id-dialog-label">
+
+          <div class="header">
+
+            <button class="prevYear"
+              aria-label="previous year"
+              aria-controls="id-dialog-label">
+              <span class="fas fa-angle-double-left fa-lg"></span>
+            </button>
+
+            <button class="prevMonth"
+              aria-label="previous month"
+              aria-controls="id-dialog-label">
+              <span class="fas fa-angle-left fa-lg"></span>
+            </button>
+
+            <span id="id-dialog-label"
+                  class="monthYear"
+                  role="status">
```

Comment:
Role status doesn't work here because it is off by default. Please change this to aria-live="polite" and remove the status role. Also, since this is the dialog title, wrap in H2